ziti/cmd/ziti/cmd/config_templates/controller.yml

```diff
@@ -119,6 +119,10 @@ web:
         # address - required
         # The public address that external incoming requests will be able to resolve. Used in request processing and
         # response content that requires full host:port/path addresses.
+        #
+        # This ziti CLI login example assumes this interface binding is 127.0.0.1:8443 and "binding: edge-management" is in the list of bound APIs below
```

Comment:
this feels like the wrong file to put `ziti` commands into. i'm curious why you'd want to add this here?

Reply:
That was the idea from the thread, to show the alignment between these settings and how they influence operation of the APIs configured here.

Reply:
We don't need ziti cli examples in the config. it's just too specific for this section. There's numerous other cli commands that need the same API - calling out ziti login just doesn't fit. Also the comment just above covers the config entry adequately.